tests/resources/mir_datasets/tonas/README.txt

```diff
@@ -0,0 +1,74 @@
+TONAS: COFLA dataset of a cappella flamenco singing 

```

Comment:
Do you need this file here? If not remove

Reply:
Done!